System/Random/MWC/Distributions.hs

```diff
@@ -70,31 +73,185 @@ normal m s gen = do
 
 
 -- | Generate truncated normally distributed random variable
-truncatedNormal
-  :: PrimMonad m
-  => Double -- ^ Standard deviation bound
+truncatedNormalRange
+  :: forall m . PrimMonad m
+  => (Maybe Double, Maybe Double) -- ^ Range to truncate over. May be a finite, or semi-finite range.
   -> Double -- ^ Mean
   -> Double -- ^ Standard deviation
   -> Gen (PrimState m)
   -> m Double
-truncatedNormal bound m s gen =
-  truncatedNormalRange (m - bound * s, m + bound * s) m s gen
+truncatedNormalRange bounds@(mmn, mmx) m s gen = do
```

Comment:
Maybe it would be easier to implement `truncatedNormalRange` in terms of `truncatedStandard`? (And of course export both)